docs/notes/bugfix-16351.md

```diff
@@ -0,0 +1,2 @@
+# After a dispatch command, the long id of the target resolves correctly 
+for shared groups and their children
```

Comment:
Unfortunately, I don't think the current release notes parser copes with wrapped lines in bug summaries!